### ISSUE TITLE
Cyclic object value error 

### ISSUE THREAD

**Issue by Bricks666** [effector/logger#112]
When I pass event or effect or what use(Unit/Store/Event and etc.) has returned me to handler of HTML5 element(like input or button) or UI component(MUI and etc.) and then start to interact with this element  I catch an error with message `Cyclic obejct value`(Firefox) or called `Converting circular structure to JSON`(Chromium) or my tab falls. Also i tried to reproduce this script on my phone. Browser was falled.

**Why i think that i bug in effector**
Because, if i pass arrow-function into handler where i will be call event/effect and etc. and don't be pass event object, it will be work fine. But i can't do like this every time when i need because it's very uncomfortable 

**Please provide the steps to reproduce and if possible a minimal demo of the problem via https://share.effector.dev, https://codesandbox.io or similar**
I couldn't to repeat it on others services

**What is the expected behavior:**
I expect It will be work like a simple handler

**Which versions of effector packages, and which browser and OS are affected by this issue? Did this work in previous versions of effector?:**
effector: 22.4.0
effector-react: 22.3.4
Firefox: 107
Chromium: 105
OS: Fedora 37
Kernel: 6.0.11

**Comment by Leniorko** [comment]
+1 on it. Can't quite understand the source of this error even with explanation from above.

Trying to use effector-logger with React and Vite. Using Todo app from this [example](https://effector.dev/docs/recipes/react/todo-creator). I prepared [codesandbox example](https://codesandbox.io/s/effector-logger-cycle-error-f81rj8) could you please check it and share the solution?

Also probably would like to have some clarification on Vite usage if error was with my config.

**Comment by Leniorko** [comment]
I have actually tried to replace all effector imports to effector-logger imports, but there is an error because I have no types in my `node_modules` for effector-logger.

Because of that, `attachLogger` to the domain also can't really work.

**Comment by Leniorko** [comment]
I found workaround by making submit event receiving void.
And moving prevent default to the handler
`src/components/tasksList/tasksList.effector.ts`
```ts
  const submit = createEvent<void>();
  // submit.watch((event) => event.preventDefault());
```
`src/components/tasksList/index.tsx`
```ts
 onClick={(event): void => {
            event.preventDefault();
            newTaskSubmit();
          }}
```

It works, but it doesn't seem like an intended way to do this.

**Comment by Leniorko** [comment]
I am actually willing to fix this bug and work on #17 (I think these issues intercepts with each other). But I need some help to set up a development environment. I never worked with libraries, so I am lacking knowledge in this area. If you can explain how do I properly test my changes in the library, I will be very thankful and will work on these bugs.

**Comment by AlexandrHoroshih** [comment]
Hello!

As of the recent '0.14.0' release, 'effector-logger' no longer includes 'effector-inspector' and integration with redux-devtools, so all related issues are gone.

This specific issue was related to these integrations and not the console logger itself.

See the release notes for details
https://github.com/effector/logger/releases/tag/v0.14.0
